testfiles/RefBldgLargeOfficeNew2004_Chicago-ReturnReset.idf

```diff
@@ -609,7 +609,7 @@
     0.0000,                  !- Direction of Relative North {deg}
     0.0000,                  !- X Origin {m}
     0.0400,                  !- Y Origin {m}
-    0.2000,                  !- Z Origin {m}
+    0.0000,                  !- Z Origin {m}
```

Comment:
Shouldn't this be -0.2 m for the basement? First floor should be 0.0.

Reply:
The basement surface z-coordinates all go from 0.0 to -2.x, so 0.0 for a zone z-origin works here.